exchanges/deribit/deribit_test.go

```diff
@@ -0,0 +1,3067 @@
+
+func TestSubmitWithdraw(t *testing.T) {
+	t.Parallel()
+	var resp *WithdrawData
+	err := json.Unmarshal([]byte(submitWithdrawalResponseJSON), &resp)
+	require.NoError(t, err)
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateRealOrders)
+	_, err = d.SubmitWithdraw(context.Background(), currencyBTC, core.BitcoinDonationAddress, "", 0.001)
+	assert.NoError(t, err)
+}
+
+func TestWSSubmitWithdraw(t *testing.T) {
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateRealOrders)
+	_, err := d.WSSubmitWithdraw(currencyBTC, core.BitcoinDonationAddress, "", 0.001)
+	assert.NoError(t, err)
+}
+
+func TestGetAnnouncements(t *testing.T) {
+	t.Parallel()
+	_, err := d.GetAnnouncements(context.Background(), time.Now(), 5)
+	assert.NoError(t, err)
+}
+
+func TestWSRetrieveAnnouncements(t *testing.T) {
+	t.Parallel()
+	_, err := d.WSRetrieveAnnouncements(time.Now(), 5)
+	assert.NoError(t, err)
+}
+
+func TestGetPublicPortfolioMargins(t *testing.T) {
+	info, err := d.GetInstrument(context.Background(), "BTC-PERPETUAL")
+	require.NoError(t, err)
+	_, err = d.GetPublicPortfolioMargins(context.Background(), currencyBTC, map[string]float64{"BTC-PERPETUAL": info.ContractSize * 2})
+	assert.NoError(t, err)
+}
+
+func TestGetAccessLog(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d)
+	_, err := d.GetAccessLog(context.Background(), 0, 0)
+	assert.NoError(t, err)
+}
+
+func TestWSRetrieveAccessLog(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d)
+	_, err := d.WSRetrieveAccessLog(0, 0)
+	assert.NoError(t, err)
+}
+
+func TestChangeAPIKeyName(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d)
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateAPIEndpoints)
+	_, err := d.ChangeAPIKeyName(context.Background(), 1, "TestKey123")
+	assert.NoError(t, err)
+}
+
+func TestWSChangeAPIKeyName(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d)
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateAPIEndpoints)
+	_, err := d.WSChangeAPIKeyName(1, "TestKey123")
+	assert.NoError(t, err)
+}
+
+func TestChangeMarginModel(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateRealOrders)
+	_, err := d.ChangeMarginModel(context.Background(), 2, "segregated_pm", false)
+	require.NoError(t, err)
+}
+
+func TestWsChangeMarginModel(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateRealOrders)
+	_, err := d.WsChangeMarginModel(2, "segregated_pm", false)
+	require.NoError(t, err)
+}
+
+func TestChangeScopeInAPIKey(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateRealOrders)
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateAPIEndpoints)
```

Comment:
These API ones can just be reduced to a single `sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateAPIEndpoints)`

Anything related to orders can use a single `sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateRealOrders)`